gallery/plot_datapoints.py

```diff
@@ -0,0 +1,122 @@
+"""
+==============
+Datapoints FAQ
+==============
+
+The :mod:`torchvision.datapoints` namespace was introduced to ``torchvision.transforms.v2``. This example showcases what
```

Comment:
Maybe we can add something along the lines of

> This is a fairly low-level topic that most users will not need to worry about: you do not need to understand the internals of datapoints to efficiently rely on the v2 transforms. It may however be useful for users trying to implement their own transforms, or work directly with datapoints.